AUTHORS.rst

```diff
@@ -0,0 +1,31 @@
+=======
+Credits
+=======
+
+Autoprotocol-Python is currently maintained by members of Strateos Inc.
+
+Here is a list of active contributors from Strateos:
+
+- Vanessa Biggers - `polarpine <https://github.com/polarpine>`_ - vanessa.biggers@strateos.com
+- Yang Choo - `yangchoo <https://github.com/yangchoo>`_ - yang.choo@strateos.com
+- Donald Dalton - `dbdalton2000 <https://github.com/dbdalton2000>`_ - donald.dalton@strateos.com
+- Varun Kanwar - `VarunKanwar <https://github.com/VarunKanwar>`_ - varun.kanwar@strateos.com
+- David Lyon - `dauglyon <https://github.com/dauglyon>`_ - david.lyon@strateos.com
+- Peter Lee - `pleaderlee <https://github.com/pleaderlee>`_ - peter.lee@strateos.com
+- Dylan Leidig - `dzleidig <https://github.com/dzleidig>`_ - dylan.leidig@strateos.com
+- Eriberto Lopez - `EribertoLopez <https://github.com/EribertoLopez>`_ - eriberto.lopez@strateos.com
+- Rhys Ormond - `rhysormond <https://github.com/rhysormond>`_ - rhys.ormond@strateos.com
+- Asuka Ota - `aota001 <https://github.com/aota001>`_ - asuka.ota@strateos.com
+- Joshua Nowak - `joshhacksthings <https://github.com/joshhacksthings>`_ - josh.nowak@strateos.com
+
+We also owe a debt to the following past contributors:
```

Comment:
Especially active past contributors we're grateful to:

Reply:
done!